packages/block-library/src/navigation-submenu/edit.js

```diff
@@ -542,6 +550,14 @@ export default function NavigationSubmenuEdit( {
 							onClick={ () => setIsLinkOpen( true ) }
 						/>
 					) }
+
+					<ToolbarButton
+						name="revert"
+						icon={ keyboardReturn }
+						title={ __( 'Revert to Link' ) }
```

Comment:
How about just 'Convert to Link'?

I'm thinking about a case where a user adds a submenu directly from the global inserter, in which case it would be a reversion.

Reply:
Sounds good!